nixos/tests/jitsi-meet.nix

```diff
@@ -16,7 +17,56 @@ import ./make-test-python.nix ({ pkgs, ... }: {
 
       networking.firewall.allowedTCPPorts = [ 80 443 ];
 
-      services.nginx.virtualHosts.server = {
+      services.nginx.virtualHosts."server" = {
+        enableACME = true;
+        forceSSL = true;
+      };
+
+      security.acme.acceptTerms = true;
+      security.acme.defaults.email = "me@example.org";
+      security.acme.defaults.server = "https://example.com"; # self-signed only
+    };
+    jwtserver = { config, pkgs, ... }: {
+      services.jitsi-meet = {
+        enable = true;
+        hostName = "jwtserver";
+      };
+      services.jitsi-videobridge.openFirewall = true;
+      services.jicofo = {
+        config = {
+          authentication = ''
+            type = JWT
+            login-url = "jwtserver"
+          '';
+        };
+      };
+      services.prosody = {
+        enable = true;
+        authentication = "token";
+        extraConfig = ''
+          app_id = "nixos";
+          app_secret = "b+bXFmEL3eIax/vjaHvcoDgQPcat8iF59QU6yq4Sb2niTXGg";
+        '';
+
+        muc = [
+          {
+            domain = "conference.jwtserver";
+            extraConfig = ''
+              modules_enabled = { "token_verification" };
+            '';
+          }
+        ];
+        virtualHosts."jwtserver" = {
+          extraConfig = ''
+            authentication = "token";
+            allow_empty_token = false;
+          '';
+        };
+      };
+
```

Comment:
This is the interface for enabling it, which I'm still cleaning up a bit.

I'm not completely sold right now because forgetting `authentication = "token";`  allows anonymous authentication again, so I may special-case 'token' authentication to add the needful to the extraConfigs.